people_tool/static/people_tool/js/controllers/CoursesController.js

```diff
@@ -84,7 +84,7 @@
                 cinfo['code'] = '';
                 cinfo['school'] = '';
             }
-
+            cinfo['section'] = course.section ? course.section : '';
```

Comment:
A little nit-picky, but maybe there's a reason, but can this line (line 87) be moved right after line 67, where other similar parameters seemed to be set?